pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1449,7 +1449,9 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
     @FieldContext(
             category = CATEGORY_SERVER,
             dynamic = true,
-            doc = "The position that replication task start at, it can be set to earliest or latest (default).")
+            doc = "The position that replication task start at, it can be set to earliest or latest (default). Note: do"
+                    + " not use \"MessageId.earliest\" because its value is \"-1:-1:-1\", which does not equals"
+                    + " \"earliest\"")
```

Comment:
Why would someone set a String value in the config to "MessageId.earliest"?
The notes about MessageId.earliest is simply confusing. It's better not to add the note at all. 
```suggestion
            doc = "The position that replication task start at, it can be set to earliest or latest (default).")
```